spec/features/groups/request_to_join_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Students Joining Groups' do
+  context 'requesting to join open group' do
+    it 'succeeds' do
+      group = FactoryGirl.create(:open_group)
+      log_in FactoryGirl.create(:student, intent: 'on_campus', draw: group.draw)
+      visit draw_group_path(group.draw, group)
+      click_on 'Request To Join'
+      expect(page).to have_content(/Membership.+created/)
+    end
+  end
+
+  context 'requesting to join a full group' do
+    it 'fails' do
+      group = FactoryGirl.create(:group)
+      log_in FactoryGirl.create(:student, intent: 'on_campus', draw: group.draw)
+      visit draw_group_path(group.draw, group)
+      click_on 'Request To Join'
+      expect(page).to have_content('Please review the errors below:')
+    end
+  end
+
+  context 'approving requests' do
+    it 'succeeds' do # rubocop:disable RSpec/ExampleLength
+      group = FactoryGirl.create(:open_group)
+      user = FactoryGirl.create(:student, intent: 'on_campus', draw: group.draw)
+      MembershipCreator.create!(group: group, user: user, status: 'requested')
```

Comment:
Yea that's what we did in all the other specs, so I'd recommend changing it here.